api/src/task/notifications/generate_notifications.py

```diff
@@ -150,9 +173,30 @@ def _send_notifications(self) -> None:
             if not container.saved_opportunities and not container.saved_searches:
                 continue
 
-            # TODO: Implement actual notification sending in future ticket
+            # Get user email from LinkExternalUser using select pattern
+            stmt = select(LinkExternalUser).where(LinkExternalUser.user_id == user_id)
+            user = self.db_session.execute(stmt).scalar_one_or_none()
+
+            if not user or not user.email:
```

Comment:
Just because we'll likely need the email for other things in the future, and it might get pulled from a few places, what if we add a property to the user table, something like:

```py
@property
def email(self) -> str | None:
     if self.login_gov_user is not None: # TODO - would need to add a relationship that fetches this
          return self.login_gov_user.email

     return None
```